### ISSUE TITLE
Grey line drawn for non-Likert types

### ISSUE THREAD

**Issue by earcanal** [expfactory/survey-generator#6]
In `style.css` this
```
/* Add gray line for Likert */

form p:before {
  content: '';
  position:relative;
  top:53px;
  left:61px;
  display:block;
  background-color:#efefef;
  height:4px;
  width:888px;
}
```
produces grey lines in non-likert form elements (e.g. `information`).  Perhaps Likert types such as `radio` could add a `likert` class to their `<p>` elements and the selector could be `form p.likert:before`?

**Comment by vsoch** [comment]
yeah! Can you do an example and possibly PR so we can take a look and discuss?

**Comment by earcanal** [comment]
Do you want an example of the issue?  I don't think I should go near the python that generates `index.html`!

**Comment by earcanal** [comment]
[Example](https://earcanal.github.io/dass-21/index.html)

[FIXME suggestion](https://github.com/earcanal/dass-21/blob/master/css/style.css#L165)

**Comment by vsoch** [comment]
Oh I see, so the visual style doesn’t change but the user can select them easily?

**Comment by earcanal** [comment]
Have a look at the example again (I've reverted `style.css` to show the issue more clearly).

**Comment by vsoch** [comment]
Sorry I don’t see how two compare the two links... for a really stupid dinosaur like myself could you post images here with a BEFORE an AFTER? Ty :)

**Comment by vsoch** [comment]
ah okay! So this picture is the BEFORE (or bug) do you have a shot of the after with the fix?

**Comment by vsoch** [comment]
here I can make one too :)

![image](https://user-images.githubusercontent.com/814322/45651354-467d8e00-ba9f-11e8-91a0-55ffc4b5e1bc.png)

**Comment by vsoch** [comment]
yes I definitely +1 here! Would you like to PR to the branch, I'll build a testing container, we can test to ensure it produces the fixed survey, then deploy?
